bin/systemd_module_migration.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# we rquire hub later on for the PR
+if ! which hub > /dev/null ; then
+  exit 1
+fi
```

Comment:
Better tell the user what is wrong on stderr than in a comment :smile: 

```suggestion
if ! which hub > /dev/null ; then
  echo "We require hub later on for the PR" >&2
  exit 1
fi
```